app/webpacker/components/RegistrationsV2/RegistrationEdit/RegistrationEditor.jsx

```diff
@@ -312,7 +312,7 @@ export default function RegistrationEditor({ competitor, competitionInfo }) {
           {(registration.payment.payment_statuses.includes('succeeded') || registration.payment.payment_statuses.includes('refund')) && (
             <Refunds
               competitionId={competitionInfo.id}
```

Comment:
Do we still need to be passing competitionId here?

Reply:
Yes, it's needed for the mutation that is actually issuing the refund